src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -351,6 +363,75 @@ def get_stats_data_for_trending_events(results):
                 else {},
             }
 
+        def fingerprint_regression(trending_event):
+            return hashlib.sha1(
+                (f'p95_transaction_duration_regression-{trending_event["transaction"]}').encode()
+            ).hexdigest()
+
+        def send_occurrence_to_plaform(found_trending_events):
+            global top_trending_transactions
+            qualifying_trends = []
+            for trend in found_trending_events:
+                if (
+                    request.GET.get("statsPeriod", None) == "14d"
```

Comment:
should this perhaps check `params["end"] - params["start"]` and see if its greater than some threshold?

Reply:
I was hoping to just use the most recent 14d trend for issue creation. the UI sets stats period to 14d by default so I'm trying to leverage that